Control/lib/api.js

```diff
@@ -189,6 +191,13 @@ module.exports.setup = (http, ws) => {
 
   // Lock Service
   http.get('/locks', lockController.getLocksStateHandler.bind(lockController));
+
+  http.put('/locks/:action/ALL',
+    minimumRoleMiddleware(Role.GLOBAL),
+    addDetectorIdMiddleware('ALL'),
```

Comment:
We should aim not to use hardcoded strings. In this case, `ALL`
The same one is used in `LockController.jsL22`. Thus, I would like you to create an enum similar to `detectorLockAction.enum.js` with this value and reuse it in both places.

Reply:
You could also reuse the `ALL` in the route as well once you extract the variable